scripts/build-images.sh

```diff
@@ -4,26 +4,13 @@
 #
 # This file is part of Invenio.
 #
-# Copyright (C) 2019 CERN.
-# Copyright (C) 2019 Northwestern University.
+# Copyright (C) 2019-2020 CERN.
+# Copyright (C) 2019-2020 Northwestern University.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 cd ${PROJECTDIR}
 
-pipfile_lock_path="./Pipfile.lock"
-
-if [ ! -f $pipfile_lock_path ]; then
-    echo "'Pipfile.lock' not found. Generate it by running 'pipenv lock'."
```

Comment:
Major: By removing the creation of the lock file it docker build will fail due to the `COPY` statement:

``` console
docker build .
Sending build context to Docker daemon  136.2kB
Step 1/8 : FROM inveniosoftware/centos7-python:3.6
 ---> d3593d479d1c
Step 2/8 : COPY Pipfile Pipfile.lock ./
COPY failed: stat /var/lib/docker/tmp/docker-builder941659066/Pipfile.lock: no such file or directory
```

I do not think copying the lock file is needed since the docker file is "install"ing the dependencies (i.e. generating a new lock file). Otherwise we should use "sync", but also commit the lock file (generating it outside to then sync makes no sense cuz it is not controlled anyway).